service/webhook.go

```diff
@@ -113,8 +121,42 @@ func WebhookEventHandler(c echo.Context, reqBody, resBody []byte) {
 			message += "\n" + "\n"
 			message += resApp.Content + "\n"
 		}
+		}
 	} else if strings.Contains(c.Request().URL.Path, "/api/transactions") {
-		//TODO
+		resApp := new(TransactionRequestApplication)
+		err := json.Unmarshal(resBody, resApp)
+		if err != nil {
+			return
+		}
+		if c.Request().Method == http.MethodPost {
+			message += "## :scroll:"
+			message += fmt.Sprintf("[入出金記録](%s/applications/%s)", "https://jomon.trap.jp", resApp.ID) + "が新規作成されました" + "\n"
+		} else if c.Request().Method == http.MethodPut {
+			message += "## :scroll:"
+			message += fmt.Sprintf("[入出金記録](%s/applications/%s)", "https://jomon.trap.jp", resApp.ID) + "が修正されました" + "\n"
+		}
+		if resApp.Amount < 0 {
+			if resApp.Group != nil {
+				message += fmt.Sprintf("- %s", resApp.Target) + "への依頼" + "\n"
+			}
+			message += fmt.Sprintf("- 支払い金額: % -s円", strconv.Itoa(resApp.Amount)) + "\n"
+		} else {
+			if resApp.Group != nil {
+				message += fmt.Sprintf("- %s", resApp.Target) + "からの振込" + "\n"
+			}
+			message += fmt.Sprintf("- 受け取り金額: %s円", strconv.Itoa(resApp.Amount)) + "\n"
+		}
+		message += fmt.Sprintf("- 関連するグループ: %s", resApp.Group.Name) + "\n"	
+		if resApp.Tags != nil {
+			message += "- タグ: "
+			tags := make([]string, len(resApp.Tags))
+			for i, tag := range resApp.Tags {
+				tags[i] = tag.Name
+			}
+
+			message += strings.Join(tags, ",")
```

Comment:
```suggestion
			message += fmt.Sprintf("- タグ: %s", strings.Join(tags, ","))
```
みたいにしたら151行目の処理をまとめられそうです

Reply:
なるほど!ありがとうございます!